app/src/main/java/com/mapbox/services/android/navigation/testapp/example/ui/ExampleViewModel.kt

```diff
@@ -65,7 +65,7 @@ class ExampleViewModel(application: Application) : AndroidViewModel(application)
 
         // Initialize navigation and pass the LocationEngine
         navigation = MapboxNavigation(getApplication(), accessToken)
-        navigation.locationEngine = locationEngine
```

Comment:
kotlin interfaces could continue to support `val`

wonder what makes us decide one way or another here

Reply:
Good question. I haven't used interfaces that also contain val. So I default to using pure interfaces. But if a case can be made to add val to an interface, I have no issues with that.